sale_medical_prescription/wizards/medical_sale_temp.py

```diff
@@ -183,7 +185,9 @@ def _to_vals(self, ):
             'origin': self.origin,
             'note': self.note,
             'team_id': self.team_id.id,
-            'payment_term': self.payment_term.id,
-            'fiscal_position': self.fiscal_position.id,
+            #  @TODO
```

Comment:
Remove comments. @lasley do we need `payment_term` and `fiscal_position`? If so, should I put it in the roadmap?